pkg/proxy/handlers/spacelister.go

```diff
@@ -265,3 +264,43 @@ func errorResponse(ctx echo.Context, err *apierrors.StatusError) error {
 	ctx.Response().Writer.WriteHeader(int(err.ErrStatus.Code))
 	return json.NewEncoder(ctx.Response().Writer).Encode(err.ErrStatus)
 }
+
+// filterUserSpaceBindings returns all the spacebindings for a given username
+func filterUserSpaceBindings(username string, allSpaceBindings []toolchainv1alpha1.SpaceBinding) (outputBindings []toolchainv1alpha1.SpaceBinding) {
+	for _, binding := range allSpaceBindings {
+		if binding.Spec.MasterUserRecord == username {
+			outputBindings = append(outputBindings, binding)
+		}
+	}
+	return outputBindings
+}
```

Comment:
yes this makes it easier! 

I think we would just not be able to report in case there are issues with the system , generating more spacebindings for the same MUR (let's say for whatever reason, a bug in Space controller or SBR controller or other parts of the system like the sandbox-cli ... ). If this is acceptable then yes, the code can be simplified.

Reply:
as decided earlier today, let's not return this error to the user. I'm not checking only for the user binding as you suggested. Plz check https://github.com/codeready-toolchain/registration-service/pull/350/commits/e8eee83db6691d4923771242f81bf01d7ac41e37